contracts/BinaryOption.sol

```diff
@@ -19,125 +18,50 @@ contract BinaryOption is IERC20, IBinaryOption {
 
     /* ========== STATE VARIABLES ========== */
 
-    string public constant name = "SNX Binary Option";
-    string public constant symbol = "sOPT";
+    string public name;
+    string public symbol;
```

Comment:
Name and Symbol will be created dynamically now as it was not user friendly to have all Options be named "sOPT"